backend-api/src/functions/common/logging/buildUserAgent.ts

```diff
@@ -0,0 +1,32 @@
+export const buildUserAgent = (userAgentHeader?: string): object => {
```

Comment:
tiny nitpick: technically this could actually stay as `string | undefined` since this helper func always gets passed a `userAgentHeader` arg in `setupLogger`. Still works as is tho so entirely non blocking


```suggestion
export const buildUserAgent = (userAgentHeader: string | undefined): object => {
```

Reply:
Yeah, true.  I think I'll leave this one.